src/workers/main.js

```diff
@@ -1,8 +1,13 @@
 import { render } from 'https://unpkg.com/preact@latest?module'
 import { html } from 'https://unpkg.com/htm/preact/index.module.js?module';
 import Counter from '../components/Counter.js';
+import WorkerSafeFrame from '../components/WorkerSafeFrame.js';
 import WorkerisedRenderer from '../utils/dom/WorkerisedRenderer.js';
 
 new WorkerisedRenderer(
-  () => render(html`<${Counter} />`, document.body)
+  () => render(html`
+    <${WorkerSafeFrame}>
```

Comment:
`WorkerSafeFrame` is a wrapper around `react-frame-component`'s `Frame` component that provides compatibility with being used in a worker's virtual DOM. This should always be used in a workerised component instead of using `Frame` directly.